eventuals/grpc/server.h

```diff
@@ -822,8 +850,19 @@ template <typename Request, typename Response>
                                                      << " and path = "
                                                      << call.context()
                                                             ->method()
-                                                     << " failed: "
-                                                     << e.what();
+                                                     << " failed: ";
+
+                                                 if (e.index() == 0) {
+                                                   EVENTUALS_GRPC_LOG(1)
+                                                       << std::get<
+                                                              0>(e)
+                                                              .what();
+                                                 } else {
+                                                   EVENTUALS_GRPC_LOG(1)
+                                                       << std::get<
+                                                              1>(e)
+                                                              .what();
+                                                 }
```

Comment:
This is the core of what you're trying to do so no need to `Raise()` above! We were doing the `Raise` before because we had a `std::exception_ptr` but now we can just do this without the `Raise()` and `Catch()`. Both places in this file please!